src/sbvr-api/sbvr-utils.ts

```diff
@@ -133,7 +135,8 @@ export interface ApiKey extends Actor {
 }
 
 export interface Response {
-	statusCode: number;
+	id?: string;
+	status: number;
```

Comment:
I still think this is uneccesary noise. 
We can continue with the statusCode for now. It's an exported interface and we don't know the implications of changing it here. Very likely open-balena-api or balena-api may import the interface and may fail on it. 

It's not neccesary to rename the property to implement batch processing, no?

Reply:
It's your decision to evaluate the impact of the change, it makes sense, but maybe not now.